.gitattributes

```diff
@@ -0,0 +1,24 @@
+# Source files
+# ============
+*.pxd		  text eol=lf
+*.py 		  text eol=lf
+*.py3 		text eol=lf
+*.pyw 		text eol=lf
+*.pyx  		text eol=lf
+*.sh      text eol=lf
+*.js      text eol=lf
+*.txt     text eol=lf
+*.css     text eol=lf
+*.html    text eol=lf
+*.md      text eol=lf
+*.yml     text eol=lf
```

Comment:
.yaml, yes. Any file which contains code that should be compiled or run need to be on this list. On that same note, AUTHORS and CONTRIBUTORS shouldn't break anything if they have CRLF vs. LF since those aren't executed. However, it won't cost us anything to add *.md to the list.